pkgs/applications/misc/keepassx/community.nix

```diff
@@ -1,5 +1,5 @@
 { stdenv, fetchFromGitHub, fetchpatch,
-  cmake, libgcrypt, zlib, libmicrohttpd, libXtst, qtbase, qttools, libgpgerror, glibcLocales
+  cmake, libgcrypt, zlib, libmicrohttpd, libXtst, qtbase, qttools, libgpgerror, glibcLocales, libyubikey, libxi, libxtst, qtx11extras
```

Comment:
According to keepassxc's wiki, they do have build instructions for mac, though I'm not sure how to add platform specific flags.

Regardless, after I added the release build flag and pushed again, the macOS X build passed.